index.html

```diff
@@ -34,5 +34,154 @@ <h2 class="welcoming">Hey there. I’m Anas
          </ul>
        <div class="angle"> <img src="images/angle.png" alt=""></div> 
    </section>
+
+   <section class="works">
+   
+   <h3 class="Section-title">My Recent Works</h3>
+      <div class="card card1">
+        <img src="images/ImgPlaholder.png" alt="">
+         <div>
+            <p>Multi-Post Stories Gain+Glory</p>
+            <ul>
+               <li>Ruby on rails</li>
+               <li>css</li>
+               <li>JavScript</li>
+               <li>html</li>
+            </ul>
+            <button type="button">See Project</button>
+         </div>
+
+      </div>
+
+      <div class="card card2">
+         <img src="images/ImgPlaholder.png" alt="">
+         <div>
+            <p>Multi-Post Stories Gain+Glory</p>
+            <ul>
+               <li>Ruby on rails</li>
+               <li>css</li>
+               <li>JavScript</li>
+               <li>html</li>
+            </ul>
+            <button type="button">See Project</button>
+         </div>
+      </div>
+      <div class="card card3">
+         <img src="images/ImgPlaholder.png" alt="">
+         <div>
+            <p>Multi-Post Stories Gain+Glory</p>
+            <ul>
+               <li>Ruby on rails</li>
+               <li>css</li>
+               <li>JavScript</li>
+               <li>html</li>
+            </ul>
+            <button type="button">See Project</button>
+         </div>
+      </div>
+      <div class="card card4">
+         <img src="images/ImgPlaholder.png" alt="">
+         <div>
+            <p>Multi-Post Stories Gain+Glory</p>
+            <ul>
+               <li>Ruby on rails</li>
+               <li>css</li>
+               <li>JavScript</li>
+               <li>html</li>
+            </ul>
+            <button type="button">See Project</button>
+         </div>
+      </div>
+      <div class="card card5">
+         <img src="images/ImgPlaholder.png" alt="">
+         <div>
+            <p>Multi-Post Stories Gain+Glory</p>
+            <ul>
+               <li>Ruby on rails</li>
+               <li>css</li>
+               <li>JavScript</li>
+               <li>html</li>
+            </ul>
+            <button type="button">See Project</button>
+         </div>
+      </div>
+      <div class="card card6">
+         <img src="images/ImgPlaholder.png" alt="">
+         <div>
+            <p>Multi-Post Stories Gain+Glory</p>
+            <ul>
+               <li>Ruby on rails</li>
+               <li>css</li>
+               <li>JavScript</li>
+               <li>html</li>
+            </ul>
+            <button type="button">See Project</button>
```

Comment:
- [ ] [OPIONAL] it would be better if you add effects button on hover and pressed in order to match the Figma design

Reply:
Hi, @youmari  as i mentioned before this something caused by webhit either use that color or either a webhint linter error comes